src/other/Audio.js

```diff
@@ -11,12 +11,14 @@
 
         this._tag = "audio";
 
-        this._source = [];
+        //this._source = [];
         this._sections = {};
     }
     Audio.prototype = Object.create(HTMLWidget.prototype);
     Audio.prototype._class += " other_Audio";
 
+    Audio.prototype.publish("source", [], "array", "Audio Source");
```

Comment:
Once properties are published the "overriding" methods need to call to "new" corresponding base class methods